backend/src/main/java/develup/submission/SubmissionResponse.java

```diff
@@ -0,0 +1,20 @@
+package develup.submission;
+
+public record SubmissionResponse(
+        Long id,
+        Long memberId,
+        Long missionId,
+        String url,
+        String comment
+) {
+
+    public static SubmissionResponse of(Submission submission) {
```

Comment:
[Request Changes]

파라미터가 1개라서 of보다 from이 적합할 것 같아요 😄

```suggestion
    public static SubmissionResponse from(Submission submission) {
```

Reply:
실수했네요 😅
찾아줘서 고마워요!